packages/botonic-plugin-contentful/tests/tools/l10n/csv-import.test.ts

```diff
@@ -8,11 +8,11 @@ import { anything, instance, mock, when } from 'ts-mockito'
 import { testManageContentful } from '../../contentful/manage/manage-contentful.helper'
 import { Locale, SPANISH } from '../../../src/nlp'
 import * as cms from '../../../src/cms'
+import { ContentId, ContentType } from '../../../src/cms'
```

Comment:
Multiple imports from `../../../src/cm`. Maybe `ContentId` and `ContentType` are the only ones needed.

Reply:
@elozano98 done in future PR after refactor. thanks!